specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,865 @@
+            "schema": {
+              "$ref": "#/definitions/PrivateEndpointConnectionListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets list of private endpoint connections on a private link scope.": {
+            "$ref": "./examples/PrivateEndpointConnectionList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describe the format of an Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateLinkScopesResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "TagsResource": {
+      "properties": {
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "A container holding only the Tags for a resource, allowing the user to update the tags on a PrivateLinkScope instance."
+    },
+    "AzureMonitorPrivateLinkScope": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties that define a Azure Monitor PrivateLinkScope resource.",
+          "$ref": "#/definitions/AzureMonitorPrivateLinkScopeProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/PrivateLinkScopesResource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "description": "An Azure Monitor PrivateLinkScope definition."
+    },
+    "AzureMonitorPrivateLinkScopeProperties": {
+      "description": "Properties that define a Azure Monitor PrivateLinkScope resource.",
+      "properties": {
+        "provisioningState": {
```

Comment:
>"provisioningState" [](start = 8, length = 19)

This will only return "Succeeded" since this is not an asynchronously created resource, right?  Do you need it?

Reply:
delete might be a long async operation. I added required property, thanks